qartod/vel3d/vel3d_qartod_gross_range_test_values.csv

```diff
@@ -1,14 +1,12 @@
 CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_instrument,{'inp': 'vel3d_k_pitch'},"{'qartod': {'gross_range_test': {'fail_span': [-300, 300]}}}",Derived from the vendor documentation,Current measurements are valid when the instrument tilts (pitch and roll) are less than 30 degrees
 CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_instrument,{'inp': 'vel3d_k_roll'},"{'qartod': {'gross_range_test': {'fail_span': [-300, 300]}}}",Derived from the vendor documentation,Current measurements are valid when the instrument tilts (pitch and roll) are less than 30 degrees
 CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_instrument,{'inp': 'vel3d_k_speed_sound'},"{'qartod': {'gross_range_test': {'fail_span': [14270, 15620]}}}",Derived from vendor documentation and instrument configuration,Assumes a valid temperature range of -4 to 40 degC for the instrument and a nominal salinity of 33 psu as programmed into the instrument
 CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_instrument,{'inp': 'vel3d_k_eastward_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-0.27771, 0.26075], 'fail_span': [-3.0, 3.0]}}}",User Gross Range based on data collected from 2014-04-19 through to 2023-12-31.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
 CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_instrument,{'inp': 'vel3d_k_northward_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-0.27916, 0.37059], 'fail_span': [-3.0, 3.0]}}}",User Gross Range based on data collected from 2014-04-19 through to 2023-12-31.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
 CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_instrument,{'inp': 'vel3d_k_upward_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-0.29644, 0.28489], 'fail_span': [-1.0, 1.0]}}}",User Gross Range based on data collected from 2014-04-19 through to 2023-12-31.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
-CE09OSPM,WFP01,01-VEL3DK000,vel3d_k_wfp_stc_instrument,{'inp': 'int_ctd_pressure'},"{'qartod': {'gross_range_test': {'suspect_span': [30.00, 515.00], 'fail_span': [0.0, 1000.0]}}}",Derived from mooring design and limits on MMP travel,Mooring design includes physical stops attached to the riser cable that limit how far up and down the MMP can travel.
```

Comment:
The revised csv looks alright. While the test is not executed, this parameter is still in downloaded datasets. I think this needs a closer look